### PR TITLE
DOC-2261 docs(crr): fix Enable the CRR setup (3.10 - 4.1)

### PR DESCRIPTION
**Pull request by runlilong**
doc-ticket: https://graphsql.atlassian.net/browse/DOC-2261

to avoid misleading that gadmin backup restore --dr is only for fresh cluster..

### CODE REVIEW

**Review by chengjie-qin — APPROVED** [review]

**Review by victorleeTG — CHANGES_REQUESTED** [review]
See my comments in the DOC ticket.

**Review by arun-TG-PM — APPROVED** [review]

**Comment by runlilong** [comment]
@victorleeTG  the pr has been updated, could you take a look?

File: modules/cluster-and-ha-management/pages/set-up-crr.adoc [hunk 1]
```diff
@@ -32,21 +32,16 @@ Make sure TigerGraph is *not* installed with a local loopback IP such as 127.0.0
...
 The following setup is needed in order to enable Cross Region Replication.
 
-=== Backup primary data
+=== Obtain the latest Backup of the primary data
 
-Use `gadmin backup` to xref:tigergraph-server:backup-and-restore:backup-cluster.adoc[].
+Retrieve the latest backup, for instance, `pr_latest_backup` from the Primary Cluster. How recent the backup is will determine how much the DR cluster lags behind and how long it will take to catch up with the Primary cluster.
 
-For how to restore a cluster from a backup that was created from another database cluster (cross cluster), refer to xref:tigergraph-server:backup-and-restore:cross-cluster-backup.adoc[].
+If there is no proper backup, use `gadmin backup create <pr_latest_backup>` to create one. For more details refer to xref:tigergraph-server:backup-and-restore:backup-cluster.adoc[].
```
Comment (victorleeTG) [inline modules/cluster-and-ha-management/pages/set-up-crr.adoc]:
```suggestion
If there is no suitable backup, use `gadmin backup create <pr_latest_backup>` to create one.
For more details refer to xref:tigergraph-server:backup-and-restore:backup-cluster.adoc[].
```

File: modules/cluster-and-ha-management/pages/set-up-crr.adoc [hunk 2]
```diff
@@ -86,20 +81,24 @@ These steps above can be skipped if these settings are already configured proper
 .Run with the latest backup from its primary cluster.
 [source.wrap,console]
 ----
-gadmin backup restore <latest-backup> --dr
+gadmin backup restore pr_latest_backup --dr
 ----
 +
-Adding the backup tag to `gadmin backup restore <backup tag> --dr` will enable CRR in a regular restore.
-For a new cluster, the latest backup parameter is not necessary.
+Adding the backup tag to `gadmin backup restore <backup tag> --dr` will enable CRR in a regular restore.
```
Comment (victorleeTG) [inline modules/cluster-and-ha-management/pages/set-up-crr.adoc]:
```suggestion
Adding `--dr` to `gadmin backup restore <backup tag>` enables CRR in a regular restore. 
```

File: modules/cluster-and-ha-management/pages/set-up-crr.adoc [hunk 3]
```diff
@@ -86,20 +81,24 @@ These steps above can be skipped if these settings are already configured proper
...
 +
 .Instead, simply run:
 [source.wrap,console]
 ----
 gadmin backup restore --dr
 ----
-+
-Excluding the tag  `gadmin backup restore --dr` will only enable CRR.
 
+Specifically, for the switching between PR and DR, before the switch, it is necessary to xref:tigergraph-server:crr:troubleshooting.adoc#_check_data_consistency_between_primary_and_dr[Check data consistency between primary and DR]
```
Comment (victorleeTG) [inline modules/cluster-and-ha-management/pages/set-up-crr.adoc]:
```suggestion
Specifically, when switching between PR and DR, before the switch, it is necessary to xref:tigergraph-server:crr:troubleshooting.adoc#_check_data_consistency_between_primary_and_dr[check data consistency between primary and DR]
```

**Review by victorleeTG — APPROVED** [review]

**Pull request merged**